src/components/MarketDetail/index.js

```diff
@@ -294,7 +293,9 @@ class MarketDetail extends Component {
                 .format(RESOLUTION_TIME.ABSOLUTE_FORMAT)}
             </div>
             {marketClosedOrFinished && (
-            <div className="marketTimer__marketClosed">{timeToResolution < 0 ? 'This market was resolved.' : 'This market was closed.'}</div>
+              <div className="marketTimer__marketClosed">
+                {`This market was ${marketResolved ? 'resolved.' : 'closed.'}`}
```

Comment:
I would move this to a constant ```const status = marketResolved ? 'resolved.' : 'closed.'``` and then
```{`This market was ${status}`}```

If tomorrow we have a different status we only have to change one variable.